engine-flink/src/main/java/org/apache/kylin/engine/flink/FlinkExecutable.java

```diff
@@ -171,35 +171,37 @@ protected ExecuteResult doWork(ExecutableContext context) throws ExecuteExceptio
             Segments<CubeSegment> mergingSeg = cube.getMergingSegments(segment);
             dumpMetadata(segment, mergingSeg);
 
+            Map<String, String> flinkConfs = config.getFlinkConfigOverride();
+            String hadoopClasspathEnv = flinkConfs.get(ENV_HADOOP_CLASSPATH_KEY);
+
             StringBuilder sb = new StringBuilder();
             if (Shell.osType == Shell.OSType.OS_TYPE_WIN) {
-                sb.append("set HADOOP_CONF_DIR=%s && export HADOOP_CLASSPATH=/usr/hdp/2.4.0.0-169/hadoop/ && %s/bin/flink run -m yarn-cluster ");
+                sb.append("set HADOOP_CONF_DIR=%s && export HADOOP_CLASSPATH=%s && %s/bin/flink run -m yarn-cluster ");
```

Comment:
should "HADOOP_CLASSPATH" also use "set" instead of "export"?

Reply:
Yes, Good catch!